frontend/src/scenes/authentication/signup/signupForm/signupLogic.ts

```diff
@@ -83,14 +84,20 @@ export const signupLogic = kea<signupLogicType>([
                 role_at_organization: '',
                 referral_source: '',
             } as SignupForm,
-            errors: ({ first_name, organization_name }) => ({
+            errors: ({ first_name }) => ({
                 first_name: !first_name ? 'Please enter your name' : undefined,
```

Comment:
nit: we call this first_name but the form is labelled `Your name` so people probably end up entering first + last.

Reply:
Agree, I will clean that up and make it more clear.